nitro-protocol/test/contracts/BatchOperator/batchDeposits.test.ts

```diff
@@ -0,0 +1,216 @@
+import {it} from '@jest/globals';
+import {expectRevert} from '@statechannels/devtools';
+import {BigNumber, Contract, Wallet, utils} from 'ethers';
+
+import {MAGIC_ADDRESS_INDICATING_ETH, getChannelId, getRandomNonce} from '../../../src';
+import {getTestProvider, setupContract} from '../../test-helpers';
+// artifacts
+import NitroAdjudicatorArtifact from '../../../artifacts/contracts/NitroAdjudicator.sol/NitroAdjudicator.json';
+import TokenArtifact from '../../../artifacts/contracts/Token.sol/Token.json';
+import BatchOperatorArtifact from '../../../artifacts/contracts/auxiliary/BatchOperator.sol/BatchOperator.json';
+import BadTokenArtifact from '../../../artifacts/contracts/test/BadToken.sol/BadToken.json';
+import {BadToken, BatchOperator, NitroAdjudicator, Token} from '../../../typechain-types';
+const provider = getTestProvider();
+
+const batchOperator = setupContract(
+  provider,
+  BatchOperatorArtifact,
+  process.env.BATCH_OPERATOR_ADDRESS!
```

Comment:
I think if you make the right edits to this file https://github.com/statechannels/go-nitro/blob/2b0bc7c5998abe021970704f448d030844a05928/nitro-protocol/test/environment.d.ts#L23 you woun't need these assertions.